quarkus/pom.xml

```diff
@@ -8,15 +8,14 @@
   <properties>
     <surefire-plugin.version>2.22.2</surefire-plugin.version>
     <maven.compiler.target>11</maven.compiler.target>
-    <quarkus.platform.version>2.6.3.Final</quarkus.platform.version>
```

Comment:
BTW, the problem you had was this. You had the Maven plugin and the BOM using different versions.